### PR TITLE
test/e2e: replace legacy scheme with client-go scheme

### PR DESCRIPTION
**Pull request by andrewsykim**
Signed-off-by: Andrew Sy Kim <kiman@vmware.com>

**What type of PR is this?**
/kind failing-test

**What this PR does / why we need it**:
Sorry I broke some tests as part of https://github.com/kubernetes/kubernetes/pull/75616. This should fix them

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/framework/create.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func (f *Framework) LoadFromManifests(files ...string) ([]interface{}, error) {
 	err := visitManifests(func(data []byte) error {
 		// Ignore any additional fields for now, just determine what we have.
 		var what What
-		if err := runtime.DecodeInto(legacyscheme.Codecs.UniversalDecoder(), data, &what); err != nil {
```
Comment (andrewsykim) [inline test/e2e/framework/create.go]: I'm a bit unsure about this change, is it safe to remove legacyscheme here? @liggitt

**Review by timothysc — APPROVED** [review]
/lgtm
/approve

**Comment by andrewsykim** [comment]
Seems like this build is stuck 

/retest

**Comment by BenTheElder** [comment]
timeouts xref #74375

**Comment by BenTheElder** [comment]
Thank you!

**Pull request merged**